lte/gateway/c/core/oai/lib/n11/SmfServiceClient.cpp

```diff
@@ -170,4 +178,76 @@ AsyncSmfServiceClient& AsyncSmfServiceClient::getInstance() {
   return instance;
 }
 
+static void handle_subs_update_location_info_ans(
+    const std::string& imsi, uint8_t imsi_length, const grpc::Status& status,
+    magma::feg::UpdateLocationAnswer response) {
+  MessageDef* message_p;
+  s6a_update_location_ans_t* itti_msg = NULL;
+
+  if (!status.ok()) {
+    std::cout << "get_subs_auth_info fails with code " << status.error_code()
+              << ", msg: " << status.error_message() << std::endl;
+
```

Comment:
Did not check in detail, but why don't you use OAILOG_...()

Reply:
Hi Lionel, I think this is mainly following what the other grpc clients have been doing. On syslog we can filter by mme service and see grpc outcomes. We need to figure this logging as a separate thing.